snap/info_test.go

```diff
@@ -493,7 +493,7 @@ func ExampleSplitSnapApp() {
 	// Output: hello-world env
 }
 
-func ExampleSplitSnapAppShort() {
+func ExampleSplitSnapApp_short() {
```

Comment:
Hmm? Did go vet crash without this change?

Reply:
This is what vet complained about:
`snap/info_test.go:496: ExampleSplitSnapAppShort refers to unknown identifier: SplitSnapAppShort`

Apparently this is a feature https://blog.golang.org/examples (although I'd say it's somewhat obscure)